src/first/Main.java

```diff
@@ -2,90 +2,40 @@
 
 import java.io.*;
 import java.util.*;
-import java.nio.file.DirectoryStream;
 
 public class Main {
-    static String extension;
-    static final String PATH = "D:\\1";
+    static String path = "D:\\1";
```

Comment:
in java usually use UPPER_CASE with const name

Reply:
done